src/plugins/path/CMakeLists.txt

```diff
@@ -1,5 +1,18 @@
+include (SafeCheckSymbolExists)
 include (LibAddMacros)
 
+set (plugin path)
+
+if (DEPENDENCY_PHASE)
+	add_definitions (-D_GNU_SOURCE)
+	safe_check_symbol_exists (euidaccess "unistd.h" TEST_EUIDACCESS)
+
+	if (!TEST_EUIDACCESS)
```

Comment:
I think you should use `NOT` instead of `!` here. As far as I know, CMake does not support the `!` operator.